src/app/stories/latest/page.tsx

```diff
@@ -0,0 +1,69 @@
+import { type GetLatestStoriesResult } from "@/app/api/stories/latest/route";
+import ArticleList from "@/components/ArticleList";
+import HomepageSection from "@/components/HomepageSection";
+import Pagination from "@/components/StoriesList/Pagination";
+import env from "@/lib/env";
+
+interface Params {
+  topic: string;
+  page_number: string;
+}
+
+export default async function StoriesLatestPage({
+  searchParams,
+}: {
+  searchParams: Params;
+}) {
+  const { topic, page_number } = searchParams;
+  const params = {
+    ...(topic ? { topic } : {}),
+    page: page_number || "1",
+  };
+
+  const { stories, total_pages } = await getStories(params);
+
+  const headerText = topic
+    ? `${topic.toUpperCase()} | LATEST`
+    : "ALL TOPICS | LATEST";
+
+  return (
+    <>
+      <div className="mx-[10%] my-10 flex flex-col gap-12">
+        <HomepageSection heading={headerText}>
+          {stories.length > 0 ? (
+            <>
+              <ArticleList articles={stories} preferHorizontal={true} />
+              <Pagination total_pages={total_pages} />
```

Comment:
I also feel like the black text on the pagination numbers might be better in white, it's a little difficult to see on the relatively dark background

Reply:
Yes, that might be better. We could make the modification in components/StoriesList/PaginationNumber.tsx. 
(I also write details for my own reference. :))